src/pudl/extract/excel.py

```diff
@@ -181,32 +135,28 @@ class GenericExtractor:
     this method should be overwritten in the dataset-specific extractor.
     """
 
-    METADATA: Metadata = None
-    """Instance of metadata object to use with this extractor."""
-
-    BLACKLISTED_PAGES = []
-    """List of supported pages that should not be extracted."""
+    METADATA: ExcelMetadata = None
```

Comment:
Once we subclassed `GenericMetadata` for this type of extractor, I had to specify its type here to get method shortcuts working in my IDE (PyCharm).

Reply:
That makes sense and seems correct to me, we want to be specifying type as much as possible.